SOLUTION4.PY

```diff
@@ -0,0 +1,14 @@
+import csv
+with open("salary.csv",'w',newline='')as f:
+     data=csv.writer(f)

```

Comment:
i want to write only" data=writer(F)". which statement i have to change? please write it.